src/app/services/mail/mail.utils.ts

```diff
@@ -287,6 +288,17 @@ export const generatePaymentConfirmationHtml = ({
     </p>
     <p>Regards,
     <br />
-    <p>${appName} team</p>   
+    ${appName} team</p>   
+  `
+}
+
+export const generatePaymentOnboardingHtml = (): string => {
+  return dedent`
+  <p>Dear Sir or Madam,</p>
+  <p>Thank you for your interest in our payments feature! <a href="${paymentConfig.landingGuideLink}">Download the file</a> to learn how to get started with payments today!</p>
+  <p>If you have any questions regarding payments, feel free to reach out to support@form.gov.sg.</p>
+  <p>Regards,
+  <br/>
+  FormSG</p>
```

Comment:
```suggestion
  <br/>
  ${appName} team</p>
```

Reply:
Aye good catch, thanks!

Reply:
Oop, needs args to be passed in as well. Done in [2f7d63a](https://github.com/opengovsg/FormSG/pull/6462/commits/2f7d63a249793621bf329b0b6b2477af3526ff95)!